src/main/java/com/scmspain/karyon/accesslog/interceptors/AccessLogInterceptor.java

```diff
@@ -87,6 +90,27 @@ private AccessLog buildAccessLogWith(HttpServerResponse<ByteBuf> response, Acces
     );
   }
 
+
+  private Optional<String> getIpAddressFromProxy(HttpServerRequest<ByteBuf> request) {
+    return Optional.ofNullable(request.getHeaders().get("X-FORWARDED-FOR"))
+      .map(ipValuesFromProxy -> ipValuesFromProxy.split(","))
+      .filter(ipsAsArray -> ipsAsArray.length > 0)
+      .map(ipsAsArray -> ipsAsArray[0].trim());
+  }
+
+  private String getIpAddressFromSocket(HttpServerRequest<ByteBuf> request) {
+
+    String ipAddress = null;
```

Comment:
I know if this is the same as in adinsertion, but,  what's the purpose of returning a null string. Maybe it's better to set it as an empty string. What do you think @oscar-raig and @aramirez-es?